lib/airbrake/resque/failure.rb

```diff
@@ -7,7 +7,7 @@ module Failure
     # @see https://github.com/resque/resque/wiki/Failure-Backends
     class Airbrake < Base
       def save
-        notice = ::Airbrake.build_notice(exception, payload)
+        notice = ::Airbrake.build_notice(exception, payload) or return
```

Comment:
Nitpick: we use `return unless (notice = ::Airbrake.build_notice(exception, payload))`